Use updated full-size favicon.

jholdstock opened the pull request:
A new favicon was generated which fills the maximum available space, rather than having a large border. This makes the DCR logo more visible, as can be seen in the following screenshot comparing dcrdata and dcrweb:

![Screenshot from 2019-03-20 07-12-12](https://user-images.githubusercontent.com/6762864/54665672-9cc79a00-4adf-11e9-887d-ef9f5d27d518.png)

dajohi commented on public/images/favicon/manifest.json:
```diff
@@ -1,18 +1,20 @@
     ],
     "theme_color": "#ffffff",
     "background_color": "#ffffff",
+    "start_url": "https://decred.org",
```
not sure this is correct.  may want just "."

dajohi commented on public/images/favicon/browserconfig.xml:
```diff
@@ -2,9 +2,11 @@
+            <square70x70logo src="/images/favicon/mstile-70x70.png?v=yHh3NA"/>
+            <square150x150logo src="/images/favicon/mstile-150x150.png?v=yHh3NA"/>
+            <square310x310logo src="/images/favicon/mstile-310x310.png?v=yHh3NA"/>
+            <wide310x150logo src="/images/favicon/mstile-310x150.png?v=yHh3NA"/>
```
this is missing.

jholdstock replied:
agh, I need to stop using `commit -a`. Fixed

chappjc commented on public/images/favicon/manifest.json:
```diff
@@ -1,18 +1,20 @@
     ],
     "theme_color": "#ffffff",
     "background_color": "#ffffff",
+    "start_url": "https://decred.org",
```
I'm not too familiar with the manifest.json, but from what I read about `start_url`, it seems like it should be a relative URL. If it is absent, it is the new desktop icon supposedly uses the location from which it was created.  Another common recommendation (MDN and developers.google.com) seems to be  `"start_url": "."`

jholdstock replied:
good spot. Fixed

chappjc replied:
I will try to test this out on a phone, but it's still a little unclear what `"."` means.  From https://developer.mozilla.org/en-US/docs/Web/Manifest:

> The URL that loads when a user launches the application (e.g. when added to home screen), typically the index. Note that this has to be a relative URL, *relative to the manifest url.*

Emphasis added by me.

jholdstock commented:
Fixed both issues. I need to go through some other repos and fix the `start_url` as well.

chappjc commented:
I think android icons might need to be refined

![image](https://user-images.githubusercontent.com/9373513/54835596-7563ff00-4c90-11e9-962a-87671b10abe3.png)

With Android's newer circle shapes, it doesn't look great.  I just used Android Asset Studio to create a set of icons for different shapes and resolutions, but not sure how packaging works:
[ic_launcher.zip](https://github.com/decred/dcrdata/files/2997250/ic_launcher.zip)

chappjc commented:
Here's another fun take on material design for our icons: 
[icons(1).zip](https://github.com/decred/dcrdata/files/2997291/icons.1.zip)
![image](https://user-images.githubusercontent.com/9373513/54836497-549ca900-4c92-11e9-8642-9de41062829b.png)

chappjc commented:
Android Studio's "Image Asset Studio" makes a more complete icon set like:

![image](https://user-images.githubusercontent.com/9373513/54837904-7a777d00-4c95-11e9-917c-af24ae439db4.png)

Although I don't know how this applies to progressive web apps as opposed to native android apps.

[ic_launcher-adaptable.zip](https://github.com/decred/dcrdata/files/2997385/ic_launcher-adaptable.zip)

chappjc commented:
Am I nitpicking or are we OK with this?  I'd rather see

![image](https://user-images.githubusercontent.com/9373513/54836533-68e0a600-4c92-11e9-85cd-ff528cf909c9.png)

or 

![image](https://user-images.githubusercontent.com/9373513/54836497-549ca900-4c92-11e9-8642-9de41062829b.png)

instead of

![image](https://user-images.githubusercontent.com/9373513/54835596-7563ff00-4c90-11e9-962a-87671b10abe3.png)

But it's not a show-stopper.

jholdstock commented:
A few changes pushed:
- Seperated the HTML elements in `extras.tmpl` and added some comments. Dealing with those was driving me crazy
- Deleted some unused icons
- Changed `manifest.json` values `start_url` and `display` as discussed on Matrix
- Use correct Decred dark blue
- Changed icons in `manifest.json` to rounded icons provided above, to improve Android support
